buildtest/main.go

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
`internal/buildtest` would be better, as this doesn't have to be a public API to users.

Reply:
Now the other test files are in direct children packages. These are NOT a Go package, but these also should be in internal packages IMO. Let's move buildtest into internal, and consider moving them later.